### ISSUE TITLE
Air Fryer support

### ISSUE THREAD

**Issue by gchrisak** [rospogrigio/localtuya#147]
Hi all,

I recently got a BlitzWolf BW-AF1 Smart Air Fryer which is tuya based. I got the datapoints and managed to use most of them as sensors and switches in HA.

Any chance this gets integrated? 

The DPS looks like this:
101: Power (On/Off)
103: Set Temperature (40 °C - 200 °C, step 5)
104: Current Temperature
105: Set time ( 1 min - 60 min, step 1)
106: Time remaining
108:  Set Temperature Unit (°C - °F)
109: Preset (standby, custom, fries, frozen, nuggets, poultry, steak, fish)
111: Start/Pause

![image](https://user-images.githubusercontent.com/65930109/98556368-a3ee7d80-22ab-11eb-9f67-71c861bf861b.png)

**Comment by postlund** [comment]
Looks we need a few time components to allow as setting some values. Should be doable in due time.

**Comment by gchrisak** [comment]
Im guessing, **input_number** for set temp, **input_datetime** for set time and **input_select** for the presets.

**Comment by postlund** [comment]
Yeah, sounds about right. Should probably be fairly simple, so if you have some coding experience you can probably implement at least one of them and push a PR 👍
